src/HDWallet.h

```diff
@@ -58,6 +58,9 @@ class HDWallet {
 
     virtual ~HDWallet();
 
+    /// Returns master key.
+    PrivateKey getMasterKey(TWCurve curve) const;
```

Comment:
Why is masterKey needed?  I don't see it used.

Reply:
not used but useful, with this I find that subkey is not compatible with slip-0010